pkgs/games/osu-lazer/default.nix

```diff
@@ -10,22 +10,29 @@
 , lttng-ust
 , numactl
 , dotnetCorePackages
+, libglvnd
+, xorg
+, udev
 }:
 
 buildDotnetModule rec {
   pname = "osu-lazer";
-  version = "2022.1101.0";
+  version = "2022.1227.0";
 
   src = fetchFromGitHub {
     owner = "ppy";
     repo = "osu";
     rev = version;
-    sha256 = "sha256-o0QHb8ebXKG0OZaYA9P/0OuOiSml3slYQ7gjyuP/ZyY=";
+    sha256 = "sha256-0iKv7H1YbTsFkKopMp9iF52mfdGaK3idurtKPNFQmGs=";
   };
 
   projectFile = "osu.Desktop/osu.Desktop.csproj";
   nugetDeps = ./deps.nix;
 
+  # TODO remove in NixOS 22.11
+  dotnet-sdk = dotnetCorePackages.sdk_6_0;
+  dotnet-runtime = dotnetCorePackages.runtime_6_0;
```

Comment:
```suggestion
```

Reply:
done in 7b28118e8509545887c84129849663d6495a162d